Warnings in 64 bit release build with Discord using MSYS2

```
ddnet/src/engine/client/discord.cpp:9:9: warning: "DISCORD_CALLBACK" redefined
    9 | #define DISCORD_CALLBACK __stdcall
      |         ^~~~~~~~~~~~~~~~
In file included from ddnet/src/engine/client/discord.cpp:6:
ddnet/ddnet-libs/discord/include/discord_game_sdk.h:19:9: note: this is the location of the previous definition
   19 | #define DISCORD_CALLBACK DISCORD_API
      |         ^~~~~~~~~~~~~~~~
ddnet/src/engine/client/discord.cpp:135:14: warning: 'virtual void CDiscord::UpdateServerInfo(const CServerInfo&, const char*)' can be marked override [-Wsuggest-override]
  135 |         void UpdateServerInfo(const CServerInfo &ServerInfo, const char *pMapName)
      |              ^~~~~~~~~~~~~~~~
ddnet/src/engine/client/discord.cpp:148:14: warning: 'virtual void CDiscord::UpdatePlayerCount(int)' can be marked override [-Wsuggest-override]
  148 |         void UpdatePlayerCount(int Count)
      |              ^~~~~~~~~~~~~~~~~
```

--- a/src/engine/client/discord.cpp
+++ b/src/engine/client/discord.cpp
@@ -5,12 +5,6 @@
 #if defined(CONF_DISCORD)
 #include <discord_game_sdk.h>
 
-#if defined(CONF_FAMILY_WINDOWS)
-#define DISCORD_CALLBACK __stdcall
-#else
-#define DISCORD_CALLBACK
-#endif
-
 typedef enum EDiscordResult(DISCORD_API *FDiscordCreate)(DiscordVersion, struct DiscordCreateParams *, struct IDiscordCore **);
 
 #if defined(CONF_DISCORD_DYNAMIC)
@@ -132,7 +126,7 @@ public:
 		m_UpdateActivity = true;
 	}
 
-	void UpdateServerInfo(const CServerInfo &ServerInfo, const char *pMapName)
+	void UpdateServerInfo(const CServerInfo &ServerInfo, const char *pMapName) override
 	{
 		if(!m_Activity.instance)
 			return;
@@ -145,7 +139,7 @@ public:
 		m_UpdateActivity = true;
 	}
 
-	void UpdatePlayerCount(int Count)
+	void UpdatePlayerCount(int Count) override
 	{
 		if(!m_Activity.instance)
 			return;

